cmp/project.py

```diff
@@ -619,28 +631,31 @@ def init_anat_project(project_info, is_new_project, debug=False):
             project_info.subject,
             anat_pipeline.input_folders,
         )
+        subject_derivatives_directory = os.path.join(
+            derivatives_directory, project_info.subject
+        )
 
     if is_new_project and anat_pipeline is not None:
         print("> Initialize anatomical project")
-        if not os.path.exists(derivatives_directory):
+        if not os.path.exists(subject_derivatives_directory):
             try:
-                os.makedirs(derivatives_directory)
+                os.makedirs(subject_derivatives_directory)
             except os.error:
-                print("... Info: %s was already existing" % derivatives_directory)
+                print("... Info: %s was already existing" % subject_derivatives_directory)
             finally:
-                print("... Info : Created directory %s" % derivatives_directory)
+                print("... Info : Created directory %s" % subject_derivatives_directory)
```

Comment:
Same here, if there's an error in the try block, 2 messages get printed, 
print("... Info: %s was already existing" % subject_derivatives_directory) first
print("... Info : Created directory %s" % subject_derivatives_directory) second.

Reply:
It should be only one message. Either "directory is created" or "directory exists"

Reply:
Addressed by commit https://github.com/connectomicslab/connectomemapper3/pull/149/commits/6c39c8e1cf3969044da41122831f9a7cf03974b4